tests/urls.py

```diff
@@ -1,7 +1,7 @@
-from django.conf.urls import include, url
+from django.urls import include, re_path
 
 from rest_framework_tus.urls import urlpatterns as rest_framework_tus_urls
 
 urlpatterns = [
-    url(r'^', include((rest_framework_tus_urls, 'rest_framework_tus_tests'), namespace='rest_framework_tus')),
+    re_path(r'^', include((rest_framework_tus_urls, 'rest_framework_tus_tests'), namespace='rest_framework_tus')),
```

Comment:
Wouldn't this work equally well?

```suggestion
    path('', include((rest_framework_tus_urls, 'rest_framework_tus_tests'), namespace='rest_framework_tus')),
```

Reply:
Probably? The deprecation warning suggested to use `re_path` and in the router itself are parts of the url variables in regex, not sure if that breaks if we use `path` here?